frontend/src/scenes/experiments/RunningTimeCalculator/runningTimeCalculatorLogic.tsx

```diff
@@ -124,13 +134,18 @@ const getSumQuery = (metric: ExperimentMetric, experiment: Experiment): TrendsQu
     } as TrendsQuery
 }
 
-const getFunnelQuery = (metric: ExperimentMetric, experiment: Experiment): FunnelsQuery => {
+const getFunnelQuery = (
+    metric: ExperimentMetric,
+    eventConfig: EventConfig | null,
+    experiment: Experiment
+): FunnelsQuery => {
     return {
         kind: NodeKind.FunnelsQuery,
         series: [
             {
                 kind: NodeKind.EventsNode,
-                event: '$feature_flag_called',
+                event: eventConfig?.event ?? '$feature_flag_called',
```

Comment:
```suggestion
                event: eventConfig?.event ?? '$pageview',
```

Reply:
This made sense to me in the case of an experiment with a reused feature flag. But I agree that `$pageview` is the right default here. Maybe we can make this smarter by checking the featue flag age.

Fixed here 8bf2ec8598d3c0f46a46be4754394e4aa269d842.